docroot/modules/humsci/hs_bugherd/config/install/bugherdapi.settings.yml

```diff
@@ -0,0 +1,11 @@
+project_key: izzlgbmd1c9xumfo9l1kxg
+disable_on_admin: 1
+api_key: t2kopaxopxvtvhqfjylxxa
```

Comment:
Is this hashed or encrypted?

Reply:
right now its not. i can . use the same method i used to encrypt the jira password